haddock-api/resources/html/Linuwial.std-theme/linuwial.css

```diff
@@ -408,7 +408,7 @@ blockquote {
 }
 
 .src {
-  background: #f4f4f4;
+  background: #eee;
```

Comment:
My take is that having one be #f7f7f7 and one be #f4f4f4 was a bit weird to begin with. Those sorts of subtle gradient distinctions are hard for users to pick up on anyway, and I thought it was clearer to take care of that as part of the PR. If you want to send the `contents-list` background to something a bit different, go for it! My taste isn't the best in these things -- I always go for simpler rather than subtler :-)